.github/workflows/build-scala.yaml

```diff
@@ -30,6 +30,10 @@ jobs:
       - uses: actions/checkout@v4
       - name: Setup Scala
         uses: japgolly/setup-everything-scala@v3.1
+        with:
+          java-version: adopt:11.0.6
```

Comment:
test on latest java - it shouldn't matter as long as you have jre11

Reply:
Updated